rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,497 @@
+# **Adding DCP support for image based-transcriptomics**
+
+## **Summary**
+
+The Human Cell Atlas (HCA) Governance Group (GG) has requested that the
+Data Coordination Platform (DCP) prioritize the incorporation of an
+image-based transcriptomics assay into the DCP. This RFC describes the
+minimum set of deliverables that must be prioritized and completed for
+the DCP to be able to (1) store and (2) process image-based single-cell
+transcriptomics experiments. The following assumptions are made:
+
+1.  Support for an image-based transcriptomics assay in the early
+    iteration phase should be added to the DCP
+
+2.  The current state of the starfish ecosystem is sufficient to
+    fulfill this need and will be selected by the AWG.
+
+3.  This instantiation will require limited scale (low hundreds of
+    datasets over several years)
+
+## **Author(s)**
+
+[Ambrose Carr](mailto:acarr@chanzuckerberg.com)
+
+[Josh Moore](mailto:j.a.moore@dundee.ac.uk)
+
+## **Shepherd**
+[Brian Raymor](mailto:braymor@chanzuckerberg.com)
+
+## **Motivation**
+
+Cellular imaging assays typically either (1) categorize cells based on
+the presence/absence of a particular set of markers, or (2) count
+structures within cells. These assays exist at variable levels of
+magnifications, including the counting of
+[<span class="underline">individual
+atoms</span>](https://www.sciencemag.org/news/2018/07/tour-de-force-researchers-image-entire-fly-brain-minute-detail)
+(Electron Microscopy (EM))
+[<span class="underline">synapses</span>](https://www.hhmi.org/news/how-to-rapidly-image-entire-brains-at-nanoscale-resolution)
+(lattice light sheet), or [<span class="underline">counting molecules
+within cells</span>](http://linnarssonlab.org/osmFISH/) (image-based
+single-cell transcriptomics and image-based proteomics).
+
+Image-based transcriptomics assays are a good candidate for DCP imaging
+support. They are relatively simple imaging assays that transform image
```

Comment:
I do not believe you make a distinction between fluorescence microscopy and other spatial transcriptomics assays like ST and Seq-well. It may be important to point out the difference and that starfish would not support the latter (to my knowledge).

Reply:
Those are sequencing based experiments, not image-based. Is that distinction too subtle?

Reply:
It probably is the right distinction, but I am not sure it will be clear to all readers given informally it is all referred to as spatial and imaging. Use your best judgment here, the solution may be being more clear here or it may be working towards a broader awareness of the distinction.